ecommerce/management/commands/refund.py

```diff
@@ -35,16 +35,14 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
```

Comment:
A little out of scope but can we make the other two arguments above required as well? And we can set the `type` of `--run` to `int` and `--amount` to `Decimal`

Reply:
👍